GameData/KerbalismConfig/Support/ROCapsules.cfg

```diff
@@ -601,26 +1002,47 @@
 {
 	@MODULE[Habitat]
 	{
-		%volume = 8.95
-		%surface = 35
+		%volume = 9	//m^3
+		%surface = 4.3267	//m^2
+		%max_pressure = 0.35
 	}
 }
 @PART[ROC-CSTCM]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
 {
 	@MODULE[Habitat]
 	{
-		%volume = 11
-		%surface = 35	//guess, very similar dimensions to Orion
+		%volume = 11	//m^3
+		%surface = 4.9461	//m^2
+		%max_pressure = 0.35
 	}
 }
+@PART[ROC-Dragon2CM|ROC-Dragon2CMLunar]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
+{
+	@MODULE[Habitat]
+	{
+		%volume = 11.8064	// m^3 ~ Calculated from the Internal Model Volume
+		%surface = 5.185	// m^2 ~ Calculated from the Internal Model Volume
```

Comment:
Not sure what you mean by "internal model volume", but not even a sphere can have that low of a surface area with that high of a volume, same as all the other previous values.